src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -254,6 +254,15 @@ public override void PrintLogo(TextWriter consoleOutput)
             consoleOutput.WriteLine();
         }
 
+        /// <summary>
+        /// Print compiler version
+        /// </summary>
+        /// <param name="consoleOutput"></param>
+        public override void PrintVersion(TextWriter consoleOutput)
+        {
+            consoleOutput.WriteLine(GetAssemblyFileVersion());
+        }
+
```

Comment:
On the other hand, the logo does print the common compiler version, so I will do that.

---
In reply to: [80316672](https://github.com/dotnet/roslyn/pull/13955#discussion_r80316672) [](ancestors = 80316672,79923998)